ci/scripts/delete_instance.sh

```diff
@@ -29,6 +29,11 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 SCRIPTDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
+. ${SCRIPTDIR}/short_circuit_pr_tasks.sh
+cd geode
+  is_source_from_pr_testable "ci" "dev-tools" "etc" "geode-book" "geode-docs" || exit 0
```

Comment:
what about README.md, LICENSE, or NOTICE in the top level? or other LICENSE/NOTICE files?

Reply:
Yup, we can add those to the list. But based on number of times those are touched, they were low-priority for testing (and I didn't think about them)